### PR TITLE
Add Hashnode to users

### PR DESCRIPTION
**Pull request by fazlerocks**
We're using Babel, and it's awesome! 💙

### CODE REVIEW

**Comment by hzoo** [comment]
Thanks for the PR @fazlerocks; we can do an AMA on hashnode sometime too

**Comment by fazlerocks** [comment]
@hzoo That would be awesome! I was about to email you for the AMA. 😀

File: images/users/hashnode.svg [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<svg id="Layer_1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 500 200">
```
Comment (Daniel15) [inline images/users/hashnode.svg]: Remove unneeded `Layer_1` id please

File: images/users/hashnode.svg [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  <style>
+    .st0{fill:#26C6DA;} .st1{fill:#607D8B;}
+  </style>
+  <title>
+    logo
```
Comment (Daniel15) [inline images/users/hashnode.svg]: Remove title

File: images/users/hashnode.svg [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<svg id="Layer_1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 500 200">
+  <style>
+    .st0{fill:#26C6DA;} .st1{fill:#607D8B;}
```
Comment (Daniel15) [inline images/users/hashnode.svg]: Just inline these?

**Comment by existentialism** [comment]
@fazlerocks thanks!

**Comment by hzoo** [comment]
@fazlerocks let me know (can dm me on twitter too)

**Pull request merged**